### PR TITLE
[OneUI] Modify Dialog basic

### PR DESCRIPTION
**Pull request by DohyungLim**
- In OneUI, a Dialog is like a TAU's Popup.
- Modify the detailed contents according to the Spec document.
- It is divided into media query according to breakpoints of mobile and tablet.
- Implementation details:
   4-1 Dialog_width
   4-2 Dialog_Height (Max Height / Scrolling)
   4-3 Dialog_Padding (Portrait / Landscape)
   4-4 Dialog_Title
   4-5 Dialog_Body text1 (Title + description)

### CODE REVIEW

File: src/css/profile/mobile/common/popup.less [hunk 1]
```diff
@@ -93,6 +88,144 @@
...
+				padding: 24px 24px 18px 24px;
+				position: sticky;
+			}
+			.ui-popup-content {
+				font-size: 16px;
+				text-align: left;
+				padding: 0px 24px 24px 24px;
+				.ui-listview {
+					border-top-width: 1px;
+					border-top-color: #CCCCCC;
```
Comment (singa2000) [inline src/css/profile/mobile/common/popup.less]: I think it's better the color define in the theme.color.less and light.variable.less.
For example,  @dropdown-menu-options-border: #cccccc;

**Comment by DohyungLim** [comment]
@TomaszLukawskiSam @HunseopJeong @singa2000 @grzegorz-czajkowski 

Comment out additional comments about the media query.

@media (min-width: 0px) and (max-width: 479px) and (min-height: 600px) and (max-height: 959px)
=> Mobile Portrait
@media (min-width: 600px) and (max-width: 959px) and (min-height: 0px) and (max-height: 479px)
=> Mobile Landscape
@media (min-width: 600px) and (max-width: 959px) and (min-height: 600px) and (max-height: 960px)
=> Tablet Portrait
@media (min-width: 960px) and (max-width: 1919px)
=> Tablet Landscape

File: src/css/profile/mobile/common/popup.less [hunk 2]
```diff
@@ -93,6 +88,144 @@
...
+		}
+	}
+}
+
+@media (min-width: 600px) and (max-width: 959px) {
+	@media (min-height: 0px) and (max-height: 479px) {
+	  	.ui-popup {
+			width: 60%;
+			margin: 0 auto;
+			background-color: rgba(255, 255, 255, 0);
```
Comment (singa2000) [inline src/css/profile/mobile/common/popup.less]: Can we define it in the light.variables.less or dark.variables.less?

File: src/js/mobile.js [hunk 3]
```diff
@@ -124,7 +124,6 @@
 			"./core/router/Router",
 			"./core/router/route/page",
 			"./core/router/route/popup",
-			"./core/router/route/popup",
```
Comment (singa2000) [inline src/js/mobile.js:127]: Why do you remove the code?
Isn't it needed?

File: src/css/profile/mobile/common/popup.less (hunk 1 above)
Comment (singa2000) [reply src/css/profile/mobile/common/popup.less]: I think you can refer to theme.color.less

//***************************************************************************
//						Popup
//***************************************************************************
@color_popup_bg: var(--popup-background); // #[color] background color
@color_popup_header_bg: var(--popup-background); // #[color] header background color
@color_popup_footer_bg: var(--popup-background); // #[color] footer background color
...

File: src/css/profile/mobile/common/popup.less [hunk 4]
```diff
@@ -93,6 +88,144 @@
...
+	@media (min-height: 600px) and (max-height: 959px) {
+		.ui-popup {
+			width: 100%;
+			margin: 0 auto;
+			background-color: rgba(255, 255, 255, 0);
+			margin-top: 40px;
+			margin-bottom: 16px;
+			padding-top: 16px;
+			.ui-popup-header {
+				font-size: 20px;
```
Comment (singa2000) [inline src/css/profile/mobile/common/popup.less]: How about the font size define in the theme.font.less?

//**************************************************************************
//                   Popup
//**************************************************************************
@font_size_popup_title_text: 50 * @unit_base;		// T121: Popup title text
@font_size_popup_title_over_sub_text: 40 * @unit_base;	// T1211: Popup title with subtitle
@font_size_popup_title_sub_text: 28 * @unit_base;	// T1212: Popup subtitle

File: src/css/profile/mobile/common/popup.less [hunk 5]
```diff
@@ -93,6 +88,144 @@
 	}
 }
 
+@media (min-width: 0px) and (max-width: 479px) {
+	@media (min-height: 600px) and (max-height: 959px) {
+		.ui-popup {
+			width: 100%;
+			margin: 0 auto;
+			background-color: rgba(255, 255, 255, 0);
+			margin-top: 40px;
```
Comment (singa2000) [inline src/css/profile/mobile/common/popup.less]: How about change px ->* px_base ?
In the tizen.less , it defined by @px_base: 1px;

File: src/js/mobile.js (hunk 3 above)
Comment (DohyungLim) [reply src/js/mobile.js:127]: This is a duplicate of the code just above.

File: src/css/profile/mobile/common/popup.less (hunk 1 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: This code should be modified to show the divider dynamically as you scroll later. 
It's not the definition you need right now.

File: src/css/profile/mobile/common/popup.less (hunk 2 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: Background-color for the border of the popup. Use it to simply look transparent. 
It's not the color inside the popup. It's not a color that's defined in the spec.

File: src/css/profile/mobile/common/popup.less (hunk 5 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: I fixed.

done.

File: src/css/profile/mobile/common/popup.less (hunk 4 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: I fixed.

done.

File: src/css/profile/mobile/common/popup.less [hunk 6]
```diff
@@ -93,6 +88,144 @@
 	}
 }
 
+@media (min-width: 0px) and (max-width: 479px) {
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: I think it would be good to apply the media query only to areas that required separation.

Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: good idea. 
however, as the existing popup works well, the spec document should be changed. 
therefore, later when the change is complete, I will separate out the common parts.

File: src/css/profile/mobile/common/popup.less [hunk 7]
```diff
@@ -5,19 +5,36 @@
 	display: none;
 	position: absolute;
 	left: 0;
-	width: 100%;
-	margin-top: 80 * @unit_base;
+    margin-bottom: 16 * @px_base;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: Need to fix indent

Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: done.

File: src/css/profile/mobile/common/popup.less [hunk 8]
```diff
@@ -5,19 +5,36 @@
...
+			width: 100%;
+			margin-top: 56 * @px_base;
+		}
 	}
 
-	@media (orientation: landscape) {
-		width: 720 * @unit_base;
-		left: 360 * @unit_base;
+	@media (min-width: 600px) and (max-width: 959px) {
+		@media (min-height: 0px) and (max-height: 479px) {
```
Comment (singa2000) [inline src/css/profile/mobile/common/popup.less]: I have a question.
Don't you check the orientation mode?

File: src/css/profile/mobile/common/popup.less (hunk 6 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: I modified.

done.

File: src/css/profile/mobile/common/popup.less (hunk 8 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: applied.

done.

File: src/css/profile/mobile/common/popup.less [hunk 9]
```diff
@@ -79,11 +102,47 @@
...
+			padding: 0 * @px_base 24 * @px_base 20 * @px_base 24 * @px_base;
+		}
+		@media (orientation: landscape) and (min-width: 960px) {
+			padding: 0 * @px_base 24 * @px_base 4 * @px_base 24 * @px_base;
+		}
+	}
+
+	.ui-listview {
+		border-top-width: 1 * @px_base;
+		border-top-color: #CCCCCC;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: Is it correct color for the border?

Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: This is the color I put in temporarily.
I plan to change it later when showing the divider dynamically as scroll.

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**